phalcon/http/cookie.zep

```diff
@@ -48,7 +48,7 @@ class Cookie implements CookieInterface, InjectionAwareInterface
 
 	protected _secure;
 
-	protected _httpOnly = true;
+	protected _httpOnly;
```

Comment:
Can we keep this to be initialized with false?

Reply:
Yeah. Otherwise it is just NULL.

Reply:
I did this when I updated the tests :| I'm unsure where it went, I'll sort it out shortly.